src/NetworkTest/testConnectivity/index.ts

```diff
@@ -252,9 +262,10 @@ function checkSubscribeToSession({ session, publisher }: PublishToSessionResults
 /**
  * Attempt to connect to the tokbox client logging server
  */
-function checkLoggingServer(OT: OT.Client, input?: SubscribeToSessionResults): Promise<SubscribeToSessionResults> {
+function checkLoggingServer(OT: OT.Client, options?: NetworkTestOptions, input?: SubscribeToSessionResults): Promise<SubscribeToSessionResults> {
   return new Promise((resolve, reject) => {
-    const url = `${getOr('', 'properties.loggingURL', OT)}/logging/ClientEvent`;
+    const loggingUrl = 'hlg.tokbox.com/prod/logging/ClientEvent';
+    const url = options && options.proxyServerUrl && `${options.proxyServerUrl}/${loggingUrl}` || `https://${loggingUrl}`;
```

Comment:
We can but it would be something like 

```
const loggingUrl = 'hlg.tokbox.com/prod/logging/ClientEvent';
const proxyUrl = getOr('', 'proxyServerUrl', options);
const url = proxyUrl ? `${proxyUrl}/${loggingUrl}` : `https://${loggingUrl}`
```

wdyt?

Reply:
OK. Let's leave it as it.